src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.js

```diff
@@ -460,6 +460,10 @@ export default class GeFormWidgetAllocation extends LightningElement {
         return `button ${this.CUSTOM_LABELS.geAddNewAllocation}`;
     }
 
+    get qaLocatorRemainingAllocationAmount() {
+        return `lightning-formatted-number Remaining Allocation Amount`;
```

Comment:
Do we want to use a custom label for the text here, or is the hard-coded "Remaining Allocation Amount" ok?

Reply:
Sravani and I chatted a bit about this. I think this is ok since the element (lightning-formatted-number) doesn't have a "static" UI facing custom label/text (it's a dynamic value based on the Donation Amount).

Long term, I'm now thinking it's probably ideal for automation to take ownership/dictate the values in these getters as they're the ultimate and only consumers.